pkg/csi_driver/utils.go

```diff
@@ -519,6 +519,7 @@ func getSidecarContainerStatus(isInitContainer bool, pod *corev1.Pod) (*corev1.C
 func isSidecarVersionSupportedForGivenFeature(imageName string, sidecarMinSupportedVersion string) bool {
 	// If the image is from our non-managed testgrid, just assume the sidecar version is supported
 	// since it's built off latest code in main
+	klog.V(4).Infof("Checking version support for image %s and minSupportedVersion %s", imageName, sidecarMinSupportedVersion)
```

Comment:
Since this is a customer facing log, I feel like this should be more understandable to the customer.  change minSupportedVersion to readable text minimum supported version, and explain checking version support for sidecar flags or something like that so they don't get confused by this log. Disregard if you will revert this after you root cause